platforms/js/build_js.py

```diff
@@ -196,12 +196,17 @@ def build_doc(self):
     parser.add_argument('--skip_config', action="store_true", help="Skip cmake config")
     parser.add_argument('--config_only', action="store_true", help="Only do cmake config")
     parser.add_argument('--enable_exception', action="store_true", help="Enable exception handling")
-    parser.add_argument('--cmake_option', action="append", help="Adds the ability to modify the list of modules")
-    args = parser.parse_args()
+    parser.add_argument('--cmake_option', action='append', help="Adds the ability to modify the list of modules")
+    parser.add_argument('--modul_option', help="Adds the ability to modify the list of modules") # change argument name
```

Comment:
Please specify the JSON file format to use here. In example, in comment before this argument.